lib/appium_lib/ios/xcuitest/command/gestures.rb

```diff
@@ -26,11 +28,12 @@ def swipe(direction:, element: nil)
         #   ```ruby
         #   scroll direction: "down"
         #   ```
-        def scroll(direction:, name: nil, element: nil, to_visible: nil, predicate_string: nil)
+        def scroll(direction:, distance: nil, name: nil, element: nil, to_visible: nil, predicate_string: nil)
```

Comment:
Can you add `# rubocop:disable Metrics/ParameterLists` at the end of this line to avoid rubocop?